packages/manager/modules/exchange/src/account/add/account-add.html

```diff
@@ -404,16 +404,42 @@
                             <div
                                 class="oui-field__control oui-field__control_m"
                             >
-                                <input
-                                    class="oui-input"
-                                    data-ng-attr-type="{{ $ctrl.shouldDisplayPasswordInput ? 'password' : 'text' }}"
-                                    id="passwordConfirmation"
-                                    name="passwordConfirmation"
-                                    required
-                                    data-ng-model="$ctrl.newAccount.passwordConfirmation"
-                                    data-ng-model-options="{ updateOn: 'blur' }"
-                                    data-ng-change="$ctrl.onPasswordConfirmationChange()"
-                                />
+                                <div class="mb-0 oui-input-overlay">
+                                    <input
+                                        class="oui-input"
+                                        data-ng-attr-type="{{ $ctrl.shouldDisplayPasswordInput ? 'password' : 'text' }}"
+                                        id="passwordConfirmation"
+                                        name="passwordConfirmation"
+                                        required
+                                        data-ng-model="$ctrl.newAccount.passwordConfirmation"
+                                        data-ng-model-options="{ updateOn: 'blur' }"
+                                        data-ng-change="$ctrl.onPasswordConfirmationChange()"
+                                    />
+                                    <button
```

Comment:
**nitpick:** Why not prefer using `ngAttr` and `ngClass` (in order to reduce code duplication)?

Reply:
Can you explain more your point ?